### PR TITLE
Design system release 23.0.2 with fixes

### PR DESCRIPTION
**Pull request by ajmaddaford**
### What is the context of this PR?
Updates to the latest design system [version](https://github.com/ONSdigital/design-system/releases/tag/23.0.2). The latest version has some breaking changes, so this pull request includes changes to resolve the issues.

### How to review 
- Check that the green tick appears where it should
- Check that address answers work

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by JamesGardiner — APPROVED** [review]

**Comment by JamesGardiner** [comment]
Addresses and green tick both seen on the test_census_thank_you_household schema

**Comment by MebinAbraham** [comment]
Does this not include the green tick that is missing from the submission panel?
Screenshot 2020-09-08 at 10 53 47

**Comment by ajmaddaford** [comment]
> Does this not include the green tick that is missing from the submission panel?
No, as the green tick wasn't there before the design system update

**Review by MebinAbraham — APPROVED** [review]

File: tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js [hunk 1]
```diff
@@ -118,8 +118,6 @@ describe("Feature: Hub and Spoke", () => {
     it("When the user returns to the Hub, Then the section should be marked as 'Completed'", () => {
       $(EmploymentTypeBlockPage.submit()).click();
       expect($(HubPage.summaryRowState("employment-section")).getText()).to.contain("Completed");
-
-      expect($(HubPage.summaryRowTitle("employment-section")).getAttribute("class")).to.contain("summary__item-title--has-icon");
```
Comment (MebinAbraham) [inline tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js:122]: Should there not be a new assert to check the icon?. I know we unit test the context but should the functional test also check it given the DS can break?

Comment (ajmaddaford) [reply tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js:122]: I originally set out to replace the css class check with a test, but decided it was for the design system to test as we don't do any visual testing in the functional tests.

**Pull request merged**